Update TLDR sorting (#8)
freight-hub/api-standards: README.md

**Pagination**: must take `page` and `pageSize` parameters and response must contains a property `items` and should contain `totalItems` for the respones (also accepted is `totalPages`)

> **Sorting**: should take `sortOrder` follow the pattern `{field_name}|{asc|desc}` as well as `sortBy` which should be a property of the resource ✅ `createdAt|asc,activatedAt|desc`

Reviewer: ```suggestion
**Sorting**: should take a `sort` parameter following the pattern `{field_name}|{asc|desc}` where `{field_name}` is a property of the resource ✅ `createdAt|asc,activatedAt|desc`
```
